src/Driver/Ftp.php

```diff
@@ -87,14 +87,14 @@ public function get($localFileName, $remoteFile, $deleteSource = false, $localFi
         $this->setLocalFilePath($localFileName, $localFile);
 
         $ftpSize = ftp_size($this->_connect(), $remoteFile) > -1;
```

Comment:
$ftpSize indicates that remote file exists ( ftp_size returns > -1) ?
Maybe some different name for a variable, $ftpRemoteExists ?